app/controllers/diseases_controller.rb

```diff
@@ -1,74 +1,6 @@
 class DiseasesController < ApplicationController
   before_action :set_disease, only: [:show, :edit, :update, :destroy]
-
-  # GET /diseases
-  # GET /diseases.json
   def index
     @diseases = Disease.all
   end
```

Comment:
before_action :set_disease, only: [:show, :edit, :update, :destroy]
can we remove this since it's redundant for now.